packages/calcite-components/src/components/input/input.scss

```diff
@@ -155,17 +155,30 @@ input {
     block-size 0,
     outline-offset 0s;
   -webkit-appearance: none;
-  &::placeholder,
-  &:-ms-input-placeholder,
-  &::-ms-input-placeholder {
-    @apply font-normal;
-    color: var(--calcite-input-placeholder-text-color, var(--calcite-color-text-3));
-  }
+
   &:placeholder-shown {
     @apply text-ellipsis;
   }
 }
 
+// Placeholder styles
+// Do not combine these style rules. The browser interprets the rule as one. Properties like -webkit can not be found in Firefox, thus the whole line is rendered as an error.
+:-ms-input-placeholder {
+  @apply font-normal;
+  color: var(--calcite-input-placeholder-text-color, var(--calcite-color-text-3));
```

Comment:
Can we use a mixin or placeholder to DRY these styles a bit? This can be done in a follow-up.